recipes/chemfiles-python/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set version = "0.6.0" %}
+{% set sha256 = "913a9ca911287e342918458619c29dfd094f4b25f7d6866048b9968fc63be207" %}
+
+package:
+  name: chemfiles-python
+  version: {{ version }}
+
+about:
+  home: http://chemfiles.github.io
+  license: MPL-v2.0
```

Comment:
Please add `license_file: LICENCE.txt` to include the license file with the package.